bolt/test/AArch64/simple-negate-ra-state.s

```diff
@@ -0,0 +1,42 @@
+# RUN: llvm-mc -filetype=obj -triple aarch64-unknown-unknown %s -o %t.o
```

Comment:
Please add a short description to the test.

nit: you can drop 'simple-' from the title.
Maybe name both tests like `negate-ra-state.s` and `negate-ra-state-incorrect.s`, so they appear next to each other.